src/de/deserializer.zig

```diff
@@ -633,19 +585,10 @@ fn Union(comptime D: type) type {
     };
 }
 
-/// Unescapes an escaped JSON string token.
-///
-/// The passed-in string must be an escaped string.
-fn unescapeString(
-    allocator: std.mem.Allocator,
-    str_token: std.meta.TagPayload(std.json.Token, std.json.Token.String),
-    slice: []const u8,
-) ![]u8 {
-    std.debug.assert(str_token.escapes == .Some);
-
-    const escaped = try allocator.alloc(u8, str_token.decodedLength());
-    errdefer allocator.free(escaped);
-
-    try std.json.unescapeValidString(escaped, slice);
-    return escaped;
+/// Like std.fmt.parseInt, but does some error conversions to better fit getty's API
```

Comment:
`parseInt` isn't public, so we should use a normal comment here.

```suggestion
// Like std.fmt.parseInt, but does some error conversions to better fit getty's API
```

Reply:
What's wrong with documenting internal functions? They won't be in the public API does anyways, and it's more fitting. Back when autodoc had internal doc mode, it also would've supported this. (I think there's still something like that)

Reply:
I personally just like using `///` for public API stuff and `//` for everything else. Just a personal preference though, we can keep it as `///` here.